packages/drivers/odsp-driver/src/odspDocumentService.ts

```diff
@@ -265,6 +246,10 @@ export class OdspDocumentService implements IDocumentService {
                     this.socketIoClientFactory(),
                 ]);
 
+            // Start refreshing join session periodically. 
+            this.refreshSessionPeriodically(requestWebsocketTokenFromJoinSession, websocketEndpoint)
```

Comment:
can we put this under a feature flag. not sure this is the right place, but would be nice to ship this off by default and then we can turn it on

Reply:
this.mc.config.getBoolean("Fluid.Driver.Odsp.refreshJoinSession");

Reply:
Using the kill switch instead of enable feature gate as per vlad suggestion as this does not seem to be a big feature change,